app/src/main/java/com/example/android/codelabs/paging/ui/SearchRepositoriesActivity.kt

```diff
@@ -148,19 +148,19 @@ class SearchRepositoriesActivity : AppCompatActivity() {
 
         val pagingData = uiState
             .map { it.pagingData }
+            // Each unique PagingData should be submitted once
             .distinctUntilChanged()
```

Comment:
@dlam `distinctUntilChanged()` is still needed bc the rest of `uiState` may change independent of it.

Reply:
Sorry I didn't read carefully enough and didn't realize we are passing individual PagingData here. How does replay work here then? I think it might be safer to just pass the reference to Flow<PagingData> through so that it can replay from cachedIn properly?